bin/spiced/src/lib.rs

```diff
@@ -82,43 +86,50 @@ pub async fn run(args: Args) -> Result<()> {
                     .context(UnableToInitializeDataSourceSnafu {
                         data_source: source,
                     })?,
-                )
+                ))
             }
             "dremio" => {
                 let dremio_auth = auth.get(source);
-                Box::new(
-                    datasource::flight::Flight::new(
-                        dremio_auth,
-                        "http://dremio-4mimamg7rdeve.eastus.cloudapp.azure.com:32010".to_string(),
-                    )
-                    .await
-                    .context(UnableToInitializeDataSourceSnafu {
-                        data_source: source,
-                    })?,
-                )
+                let Some(url) = ds.params.get("url") else {
```

Comment:
Now that this comes from params we'll need to update `spice dataset configure` to set it. For now, it can probably just default to this URL without prompting the user when Dremio datasource is selected.

Reply:
I'm working on a PR to fix this